### PR TITLE
Project Columns

### PR DESCRIPTION
**Pull request by davidcostadev**
print-central

### CODE REVIEW

File: app/assets/stylesheets/new_board/_screen.scss [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    background: transparent;
+    border-width: 0;
+    padding: 4px 11px;
+    display: inline-flex;
+    justify-content: center;
+    align-items: center;
+    border-radius: 2px;
+
+    &:hover {
+      background-color: rgba(0, 0, 0, 0.3);
```
[the remark this answers is not in the document]
Comment (ViniciusAlmeidaSilva) [reply app/assets/stylesheets/new_board/_screen.scss]: Yes, it looks better! Done

File: app/assets/stylesheets/new_board/_project_board.scss [hunk 2]
```diff
@@ -0,0 +1,6 @@
+.Project-board {
```
[the remark this answers is not in the document]
Comment (ViniciusAlmeidaSilva) [reply app/assets/stylesheets/new_board/_project_board.scss]: Ok! Done

File: app/assets/javascripts/components/projects/ProjectBoard.js [hunk 3]
```diff
@@ -13,16 +14,40 @@ class ProjectBoard extends React.Component {
     }
 
     return (
-      <div>{this.props.project.name}</div>
+      <div className="Project-board">
```
[the remark this answers is not in the document]
Comment (ViniciusAlmeidaSilva) [reply app/assets/javascripts/components/projects/ProjectBoard.js]: Done

File: app/assets/stylesheets/_variables.scss [hunk 4]
```diff
@@ -76,3 +76,8 @@ $sidebar-fg-selected:           #bbb;
 $sidebar-fg-hover:              #fff;
 $sidebar-settings-fg:           $sidebar-fg-selected;
 $sidebar-settings-fg-selected:  #fff;
+
+/* Column */
+
+$btn-close-hover:               rgba(0, 0, 0, 0.3);
```
Comment (talyssonoc) [inline app/assets/stylesheets/_variables.scss]: It would be better to rename these variables to be more specific. Something like `$column-btn-close-hover`

Comment (ViniciusAlmeidaSilva) [reply app/assets/stylesheets/_variables.scss]: Ok!

**Pull request merged**